client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -70,48 +72,50 @@ export default React.createClass( {
 		}
 
 		return (
-			<div className="main main-column" role="main">
-				<SidebarNavigation />
-				<StatsNavigation section="insights" site={ site } />
-				<div id="my-stats-content">
-					<PostingActivity streakList={ streakList } />
-					<LatestPostSummary site={ site } />
-					<TodaysStats
-						site={ site }
-						summaryData={ summaryData }
-						path={ '/stats/day' }
-						insights={ true }
-					/>
-					<AllTime allTimeList={ allTimeList } />
-					<MostPopular insightsList={ insightsList } />
-					<div className="stats-nonperiodic has-recent">
-						<div className="module-list">
-							<div className="module-column">
-								<Comments
-									path={ 'comments' }
-									site={ site }
-									commentsList={ commentsList }
-									followList={ followList }
-									commentFollowersList={ commentFollowersList } />
-								{ tagsList }
-							</div>
-							<div className="module-column">
-								<Followers
-									path={ 'followers' }
-									site={ site }
-									wpcomFollowersList={ wpcomFollowersList }
-									emailFollowersList={ emailFollowersList }
-									followList={ followList } />
-								<StatsModule
-									path={ 'publicize' }
-									moduleStrings={ moduleStrings.publicize }
-									site={ site }
-									dataList={ publicizeList } />
+			<ReduxProvider store={ store }>
```

Comment:
@youknowriad the stats controller is an _exciting_ place.  Typically I just match the syntax that is in place of the file, and reserve ES6 updates for separate PRs.